src/main/java/ru/skypro/lessons/springboot/weblibrary/pojo/Employee.java

```diff
@@ -1,31 +1,61 @@
 package ru.skypro.lessons.springboot.weblibrary.pojo;
 
+import jakarta.persistence.*;
+
+@Entity
+@Table (name = "employee")
 public class Employee {
-    private final String name;
-    private int salary;
+@Id
+@GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+    private String name;
+    private Integer salary;
+    private static int idGeneration = 1;
+    @ManyToOne(cascade = CascadeType.PERSIST, fetch = FetchType.LAZY)
```

Comment:
Давай тут поменяем CascadeType на ALL
```suggestion
    @ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
```